stacks/knot-cloud-core/README.md

```diff
@@ -0,0 +1,37 @@
+# knot-cloud-core
+
+### Configure DNS
+
+This stack uses [Traefik](https://traefik.io) as a reverse proxy for the services and requires that you configure your DNS server to point, at least, the **things**, **users** and **bt** subdomains to the machine where the stack is being deployed, so that it can route the requests to the appropriated service. It is possible to configure it to route by path or port, but instructions for that won't be provided here for brevity.
+
+If you don't have a domain or can't configure the main DNS server, you can configure a test domain in your machine before proceeding. Either setup a local DNS server, e.g. [bind9](https://wiki.debian.org/Bind9), or alternatively update your hosts file to include the following addresses:
+
+```
+127.0.0.1   things
+127.0.0.1   users
+127.0.0.1   bt
+```
+
+On Windows, the hosts file is usually located under `c:\Windows\System32\Drivers\etc\hosts`. On Unix systems, it is commonly found at `/etc/hosts`. Regardless of you operating system, administrator or super user privileges will be required.
+
+### Deploy: stage 1
+
+Stage 1 contains the core services. The next sections provide the instructions to configure and deploy them. Whenever a configuration file is mentioned, it refers to a file found at `stacks/prod/env.d`, for production mode, or `<path>/stack/dev/env.d`, for development mode.
```

Comment:
The path should be updated & we don't have neither `prod` for `knot-cloud-core` nor a `dev` being generated in `knot-cloud init knot-cloud-core`. :grimacing: 

```suggestion
Stage 1 contains the core services. The next sections provide the instructions to configure and deploy them. Whenever a configuration file is mentioned, it refers to a file found at `stacks/knot-cloud-core/dev/env.d` or `<path>/stack/env.d`.
```

Also related to this, why isn't the default path (for `knot-cloud init <stack>`) `knot-cloud/stacks/knot-cloud{-core}/dev/`? If the reason is that the `/dev` dir already exists, why not just clone the repos instead of also generating configs? It gets quite confusing to have `stack` and `stacks` on the same tree level.

Reply:
About your questions:

1. I don't really know. At least for me, the default should be the developer's `home` directory. This would avoid the `knot-cloud` directory becoming a mess.
2. The pre-existing `/dev` dir exists to avoid to emphasize that stack isn't suitable for development.

I agree with you that it gets confusing but creating inside the `dev` directory wouldn't improve that. I think the developer shouldn't use this directory as their workspace (only if really want).